subsys/bluetooth/controller/hci/hci.c

```diff
@@ -129,6 +129,18 @@ static struct net_buf *cmd_status(uint8_t status)
 	return bt_hci_cmd_status_create(_opcode, status);
 }
 
+static struct net_buf *cmd_complete_status(uint8_t status)
+{
+	struct net_buf *buf;
+	struct bt_hci_evt_cc_status *ccst;
+
+	buf = bt_hci_cmd_complete_create(_opcode, sizeof(*ccst));
```

Comment:
@andrzej-kaczmarek why are you not using `hci_cmd_complete()` here instead of duplicating the code?

Reply:
Could you check both options and see if there´s an impact to code size?

Reply:
tbh I just did not think about it, but have just checked both versions and current one is a bit smaller:
`FLASH:      127124 B         1 MB     12.12%` (current)
`FLASH:      127588 B         1 MB     12.17%` (fixed)
numbers above are from `hci_usb`